signac_dashboard/templates/login.html

```diff
@@ -0,0 +1,25 @@
+{% extends "layout.html" %}
+
+{% block title %}{% endblock %}
+{% block subtitle %}{% endblock %}
+{% block view_options %}{% endblock %}
+{% block module_selector %}{% endblock %}
+{% block searchbar %}{% endblock %}
+
+{% block panels %}
+
+<div class="container column is-8">
+  <form method="POST" action="/login" class="box">
+    <h4 class="title">Logged out</h4>
```

Comment:
I would pick a phrase that differentiates between the states of "Logged out" and "Not logged in." This heading makes it sound like the user has explicitly clicked a button to "log out," rather than the state of "not (having yet) logged in."

Reply:
Good point